### PR TITLE
Move the lint and i18n-check after docker image in CI

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -140,12 +140,6 @@ workflows:
   untagged-build:
     jobs:
       - install
-      - lint:
```
Comment (cpanato) [inline .circleci/config.yml]: this does not change anything, and we already doing the build and the lint in parallel so we are good

Comment (jwilander) [reply .circleci/config.yml]: Yeah, I realized that after I saw it build. We'll just use this PR to test and we can close it after

**Comment by jwilander** [comment]
@bradjcoughlin it worked, the hash matches the latest commit that has the failed linting

Screen Shot 2019-09-25 at 14 31 13

**Comment by hanzei** [comment]
@jwilander Will you continue working on this?

**Comment by jwilander** [comment]
No this isn't needed anymore

**Pull request closed without merging**